frontend/src/metabase/admin/settings/components/Swag/SwagModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { useLocalStorage } from "react-use";
+import { t } from "ttag";
+
+import { getCurrentUser } from "metabase/admin/datamodel/selectors";
+import { useSelector } from "metabase/lib/redux";
+import { Button, Flex, Modal, type ModalProps, Text } from "metabase/ui";
+
+import { SWAG_51_LOCAL_STORAGE_KEY, SWAG_LINK } from "./constants";
+
+export const SwagModal = (props: Pick<ModalProps, "opened" | "onClose">) => {
+  const user = useSelector(getCurrentUser);
+  const url = user.email ? `${SWAG_LINK}?email=${user.email}` : SWAG_LINK;
```

Comment:
did we decide against sending the instance uuid?

Reply:
It was mentioned, but I don't think we are going to bother.

https://metaboat.slack.com/archives/C07NW0WQG30/p1727404321295939?thread_ts=1727402929.582489&cid=C07NW0WQG30